src/pixie/fileformats/svg.nim

```diff
@@ -563,41 +596,26 @@ proc draw(img: Image, node: XmlNode, ctxStack: var seq[Ctx]) =
 proc decodeSvg*(
   data: string, width = 0, height = 0
 ): Image {.raises: [PixieError].} =
-  ## Render SVG file and return the image. Defaults to the SVG's view box size.
+  ## Render SVG file and return the image.
+  ## Defaults to the SVG's width and height or view box size.
   try:
     let root = parseXml(data)
     if root.tag != "svg":
       failInvalid()
 
-    let
-      viewBox = root.attr("viewBox")
-      box = viewBox.split(" ")
-      viewBoxMinX = parseInt(box[0])
-      viewBoxMinY = parseInt(box[1])
-      viewBoxWidth = parseInt(box[2])
-      viewBoxHeight = parseInt(box[3])
-
-    var rootCtx = initCtx()
-    rootCtx = decodeCtx(rootCtx, root)
-
-    if viewBoxMinX != 0 or viewBoxMinY != 0:
-      rootCtx.transform = rootCtx.transform * translate(
-        vec2(-viewBoxMinX.float32, -viewBoxMinY.float32)
-      )
-
-    if width == 0 and height == 0: # Default to the view box size
-      result = newImage(viewBoxWidth, viewBoxHeight)
-    else:
+    block:
+      var
```

Comment:
I think we need an if statement if we are using Rect formed by view box or rect from by width and height. I think this can be simpler?